org.bndtools.templating.gitrepo/resources/processed/org/bndtools/templating/jgit/initialrepos.txt

```diff
@@ -1 +1 @@
-bndtools/workspace;branch=origin/${version;==;${base.version}}
```

Comment:
This is wrong. We must have version specific branches so older versions of Bndtools don't stop working when we update master for newer branches. We just forgot to update the repo for 5.0.0. (Need to add this task to the release process checklist.)